doc/api/errors.md

```diff
@@ -1004,6 +1004,22 @@ Used when a callback is called more then once.
 can either be fulfilled or rejected but not both at the same time. The latter
 would be possible by calling a callback more then once.
 
+<a id="ERR_NET_INVALID_LISTEN_ARG"></a>
+### ERR_NET_INVALID_LISTEN_ARG
```

Comment:
I used the `ERR_INVALID_OPT_VALUE` error instead of creating a new error.
I amended the change and pushed -f to the current pull request.